src/main/java/net/neoforged/neoforge/event/entity/living/LivingEntityUseItemEvent.java

```diff
@@ -43,8 +57,16 @@ public void setDuration(int duration) {
      *
      */
     public static class Start extends LivingEntityUseItemEvent implements ICancellableEvent {
+        /**
+         * @deprecated Use {@link Start#Start(LivingEntity, ItemStack, InteractionHand, int) the hand sensitive version} as this version provides wrong hand information
+         */
+        @Deprecated(forRemoval = true)
```

Comment:
Add the `since` part to this annotation.

Reply:
```suggestion
        @Deprecated(since = "1.21.5", forRemoval = true)
```